modules/auxiliary/scanner/ftp/easy_file_sharing_ftp.rb

```diff
@@ -0,0 +1,112 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::Ftp
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Easy File Sharing FTP Server Version 3.6 and Earlier Directory Traversal Information Disclosure',
```

Comment:
After installing the 3.5 version. Go to the about box. They mentioned that the installed version is 3.6